cmd/loadtest/loadtest.go

```diff
@@ -316,19 +316,21 @@ func completeLoadTest(ctx context.Context, c *ethclient.Client, rpc *ethrpc.Clie
 	var err error
 	finalBlockNumber, err = waitForFinalBlock(ctx, c, rpc, startBlockNumber, startNonce, currentNonce)
 	if err != nil {
-		log.Error().Err(err).Msg("there was an issue waiting for all transactions to be mined")
+		log.Error().Err(err).Msg("There was an issue waiting for all transactions to be mined")
 	}
-	endTime := time.Now()
-	startTime := loadTestResults[0].RequestTime
-	log.Debug().Uint64("currentNonce", currentNonce).Uint64("final block number", finalBlockNumber).Msg("got final block number")
+	if len(loadTestResults) > 0 {
```

Comment:
lgtm, although i do prefer an early return here haha

Reply:
Oh you're right, I also prefer this approach. It's implemented :)